### DISCUSSION TITLE
Is it possible to get the output from a tool like CodeInterpreter using the new Agents.OpenAI framework

### DISCUSSION THREAD

**Discussion by rwhitworth127** [microsoft/semantic-kernel#6114] [Q&A] [answered]
I have run the samples to use the OpenAIAssistantAgent to retrieve the messages from the assistant on the thread. However, if I use the Azure Assistants Playground (or the OpenAI assistants interface at OpenAI), I can see where the tool itself is generating messages (for example, the code it ran and the results). 
Is this something I can access currently with the framework? If not, are there plans to do this? Is it even exposed at a level I could write this myself?

**Comment by sophialagerkranspandey** [comment, marked as answer]
This is a great feature enhancement, we'll turn this into an issue. Thanks.
